### ISSUE TITLE
Can I use it in a Kotlin Multiplatform project?

### ISSUE THREAD

**Issue by bhaveshtandeldigitas** [cloudinary/cloudinary_kotlin#104]
I'm trying to add it to a KMP project, but the dependency is not resolved. If I added directly to the Android project I can access all clases of the SDK. 

Is there a way of making it work for KMP?

**Comment by tommyg-cld** [comment]
@bhaveshtandeldigitas so our SDK has not been configured to work with KMP but this is something we'll look into. 

for now please add it directly as you mentioned.

**Comment by mboudraa** [comment]
Yes please. Generating urls on the multiplatform side is super important. Especially since there's nothing platform specific. 
It should be compatible by simply adding more targets

**Comment by tommyg-cld** [comment]
@mboudraa thanks for your feedback, i have passed it internally and will let you know if anything changes on our end.
